astro/src/content/docs/release-notes/index.mdx

```diff
@@ -47,11 +47,118 @@ import { YouTube } from '@astro-community/astro-embed-youtube';
 
 Looking for release notes older than 1.44.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications) <span class="not-prose no-underline"><a class="ml-2" href="/docs/releases.xml"><i class="fas fa-xs fa-rss text-orange-700 text-2xl" width="50px" /></a></span>
 
+<ReleaseNoteHeading version='1.60.0' releaseDate='September 10, 2025' name='Prompt Param Piranha' />
+
+### Security
+More information coming soon.
+
+### New
+* <Issue issue="2208">
+    FusionAuth supports the OIDC _prompt_ parameter! This enables various use cases, such as silent authentication, as well requesting re-authentication and consent prompting.
+
+    Thanks to all of the community members that have helped us define this requirement, and waited patiently for it! 🎉
+  </Issue>
+
+### Fixed
+* <Issue issue="2691">
+    When decommissioning a license in a FusionAuth cluster, the change may not be reflected in all cluster nodes immediately. This can provide intermittent and inconsistent access to licensed features, depending on which node is handling a request.
+  </Issue>
+
+* <Issue issue="2887">
+    A user can be inadvertently shown an error when a tenant is using Advanced Threat Detection to send rate limit emails for two-factor authentication as the rate limit is exceeded by the user's action.
+
+    The user will now be shown the correct message when this condition is met.
+  </Issue>
+
+* <Issue issue="2936">
+    When a user completes a forgot password workflow, the failed login count will be reset.
```

Comment:
Note I updated the copy here to not say Locked. We have a separate configuration for locking an account to prevent login using "User actions" - but that is something else. 

The thing we changed here was to reset the failed login count which is part of the rate limiting system. Feel free to update the copy, but I didn't want to make it sound like it was the account "locking" which is the user action, because that was already being removed on a password reset.